test/EFCore.PG.FunctionalTests/Query/UnaccentQueryNpgsqlTest.cs

```diff
@@ -0,0 +1,156 @@
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.TestUtilities;
+using Microsoft.Extensions.DependencyInjection;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Infrastructure;
+using Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query
+{
+    /// <summary>
+    /// Provides unit tests for the pg_trgm module operator and function translations.
+    /// </summary>
+    /// <remarks>
+    /// See: https://www.postgresql.org/docs/current/pgtrgm.html
+    /// </remarks>
+    public class UnaccentQueryNpgsqlTest : IClassFixture<UnaccentQueryNpgsqlTest.UnaccentQueryNpgsqlFixture>
```

Comment:
After merging into the main provider, the tests in this class can simply go into NorthwindFunctionsQueryNpgsqlTest - there are quite a few names with accents in the Northwind data set.

Reply:
I tried to put the tests into the NorthwindDbFunctionsQueryNpgsqlTest class but I wasn't sure how to add the postgres extension. So, for now I left it in its own test class.

Reply:
You should be able to do this by adding it to NpgsqlNorthwindContext, [just like we already do with uuid-ossp](https://github.com/npgsql/EFCore.PG/blob/7fee80c8e96a2314ed62d011d1e3e20f17189728/test/EFCore.PG.FunctionalTests/TestModels/Northwind/NorthwindNpgsqlContext.cs#L14).